src/app/Http/Controllers/quizyController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use App\Http\Requests\HelloRequest;
+use Illuminate\Support\Facades\Validator;
+
+class quizyController extends Controller
```

Comment:
頭文字を大文字にしてあげるのが規則かと思います
```suggestion
class QuizyController extends Controller
```